services/ui-src/src/utils/reports/entities.test.ts

```diff
@@ -1,5 +1,9 @@
 import { getAddEditDrawerText, getFormattedEntityData } from "./entities";
+// constants
```

Comment:
most of the updates to this file are rearranging and reorganizing tests. see comment below for where new changes start